new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTraefikLoadBalancer.java

```diff
@@ -0,0 +1,353 @@
+      Map<String, Integer> clusterNameMsPortMap = new HashMap<>();
+      clusterNameMsPortMap.put("cluster-1", 8001);
+      logger.info("Creating ingress resource for domain {0} in namespace {1}", domainUid, domainNamespace);
+      createTraefikIngressForDomainAndVerify(domainUid, domainNamespace, 0, clusterNameMsPortMap, true);
+
+      logger.info("Getting node port for admin server default channel");
+      int serviceNodePort = assertDoesNotThrow(()
+          -> getServiceNodePort(domainNamespace, adminServerPodName + "-external", "default"),
+          "Getting admin server node port failed");
+
+      logger.info("Deploying application {0} in domain {1} cluster target cluster-1",
+          clusterViewAppPath, domainUid);
+      ExecResult result = DeployUtil.deployUsingRest(K8S_NODEPORT_HOST,
+          String.valueOf(serviceNodePort),
+          ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT,
+          "cluster-1", clusterViewAppPath, null, domainUid + "clusterview");
+      assertNotNull(result, "Application deployment failed");
+      logger.info("Application deployment returned {0}", result.toString());
+      assertEquals("202", result.stdout(), "Deployment didn't return HTTP status code 202");
+
+      bindDomainName(domainUid);
+    }
+
+    // verify load balancing works when 2 domains are running in the same namespace
+    for (String domainUid : domains) {
+      verifyLoadbalancing(domainUid, replicaCount, managedServerNameBase);
+    }
+  }
+
+  private void verifyLoadbalancing(String domainUid, int replicaCount, String managedServerNameBase) {
+    //access application in managed servers through traefik load balancer
+    logger.info("Accessing the clusterview app through traefik load balancer");
+    String curlRequest = String.format("curl --silent --show-error --noproxy '*' "
+        + "-H 'host: %s' http://%s:%s/clusterview/ClusterViewServlet",
+        domainUid + "." + domainNamespace + "." + "cluster-1" + ".test", K8S_NODEPORT_HOST, nodeportshttp);
+    List<String> managedServers = new ArrayList<>();
+    for (int i = 1; i <= replicaCount; i++) {
+      managedServers.add(managedServerNameBase + i);
+    }
+    assertThat(verifyClusterMemberCommunication(curlRequest, managedServers, 20))
+        .as("Verify applications from cluster can be acessed through the traefik loadbalancer.")
+        .withFailMessage("application not accessible through traefik loadbalancer.")
+        .isTrue();
+
+    boolean hostRouting = false;
+    //access application in managed servers through traefik load balancer and bind domain in the JNDI tree
+    logger.info("Accessing the clusterview app through traefik load balancer");
+    String curlCmd = String.format("curl --silent --show-error --noproxy '*' "
+        + "-H 'host: %s' http://%s:%s/clusterview/ClusterViewServlet?domainTest=" + domainUid,
+        domainUid + "." + domainNamespace + "." + "cluster-1" + ".test", K8S_NODEPORT_HOST, nodeportshttp);
```

Comment:
You can change this to: domainUid + "." + domainNamespace + ".cluster-1.test", K8S_NODEPORT_HOST, nodeportshttp);

Reply:
fixed